mitodl/mit-open
# Text Input + Select components

### What are the relevant tickets?
Closes #https://github.com/mitodl/hq/issues/4020

### Description (What does it do?)
Adds input and select components:
- TextField (text input + labels, helptext, errortext) [storybook](https://ordinary-kitty.surge.sh/?path=/docs/smoot-design-textfield--docs)
- Input (text input only) [storybook](https://ordinary-kitty.surge.sh/?path=/docs/smoot-design-input--docs)
- SelectField (select input + labels, helptext, errotext) [storybook](https://ordinary-kitty.surge.sh/?path=/docs/smoot-design-selectfield--docs)
    - Select (select input, no label, etc) also exported.

### Screenshots (if appropriate):
See storybooks.

### How can this be tested?
1. docker compose exec watch yarn storybook, view stories:
- http://localhost:6006/?path=/docs/smoot-design-textfield--docs
- http://localhost:6006/?path=/docs/smoot-design-input--docs
- http://localhost:6006/?path=/docs/smoot-design-selectfield--docs
2. Click around app. It should still look good.
    - In particular, the dialogs for editing/creating learningpaths / userlists look better.
